commands/shortenCommand.go

```diff
@@ -21,28 +21,39 @@ func shortenCommand(ctx context.Context, s *discordgo.Session, m *discordgo.Mess
 	}
 
 	params := strings.Split(m.Content, " ")
-	if len(params) == 1 {
-		s.ChannelMessageSend(m.ChannelID, "Missing argument: original-url")
+	if len(params) < 3 {
+		s.ChannelMessageSend(m.ChannelID, "Missing parameter: shortened-url")
```

Comment:
error message here doesnt make sense anymore

Reply:
also surely more detailed syntax errors are nice

Reply:
!shorten https://url.com slug
!shorten delete slug

I need to add that to register commands